### ISSUE TITLE
version 6.3 compatible?

### ISSUE THREAD

**Issue by ameeno** [ayufan/pve-patches#61]
others have said does not work on 6.2.11 or 6.2.9

i just installed pve 6.3.

how can this work? i want differential backups.

**Comment by dkmn-123** [comment]
Me too...

**Comment by ayufan** [comment]
I was looking recently how this could be ported, and likely there's a way,
but don't expect this to be likely supported for longer than maybe 6.3, as:
- 6.3 has a number of changes how the settings are handled and stored
- 6.3 also has a pretty decent PBS that actually is superior if we talk
about incremental backups (not differential)

So, I'm slowly thinking that we should be switching to PBS :)

**Comment by ameeno** [comment]
PBS looks good but i belive we would need a separate machine to handle the pbs side of things right? this doesn't really solve the incremental backup/snapshot issue that exists directly within a single proxmox host.

unless you want to have a VM that holds pbs. but that's risky (backups inside the VM had image).

not really sure, is there a good guide to use pbs and proxmox on a single host?

**Comment by dkmn-123** [comment]
I have tried to switch to pbs running as an lxc container (debian) inside my only machine.
I have problem that it does not let the datastore disks sleep.
https://forum.proxmox.com/threads/disks-can-not-sleep.80042/

**Comment by ayufan** [comment]
Unsure how you configured that, but I see my HDD sleeping.

I do:
- LVM does not touch HDD drivers
- I do manually mount HDD
- Only datastore is configured

**Comment by dkmn-123** [comment]
datastore disk is sleeping?

**Comment by ayufan** [comment]
Yes. I would assume that `vgs` is waking up the disks for you, at least it
was in my case.

I added them to the list of ignored ones in `lvm.conf`.

**Comment by dkmn-123** [comment]
We speak of proxmox backup server as lxc debian container inside proxmox ve or normal VM?

**Comment by ayufan** [comment]
I run PBS alongside Proxmox VE, so not in LXC or QEMU. LXC would not make a
difference here either way.

I look always at `/proc/sys/vm/block dump` to see what is waking spinning
disk.
